syfertext/pipeline/subpipeline.py

```diff
@@ -144,6 +144,9 @@ def __call__(
         # Language object to create a DocPointer object
         if self.client_id != self.owner.id:
 
+            # set the owner of the doc object as the current worker
```

Comment:
I suggest to do this on line 135 above. We assign the owner regardless of where the subpipeline is.

I actually missed owner assignement for Doc, It was written here https://github.com/OpenMined/SyferText/blob/63932bf418dcf0c42e180afdbe049cb5937ed99f/syfertext/tokenizer.py#L138-L141

That this should be done. Thank you for signaling this

Reply:
Yes, I noticed that earlier 👍

Reply:
Great, would then add a test for that please

Reply:
Sure 👍